packages/deb/control

```diff
@@ -0,0 +1,25 @@
+Package: libcapstone-dev
+Source: capstone
+Version: <version-placeholder>
+Architecture: amd64
+Maintainer: Rot127 <unisono@quyllur.org>
+Original-Maintainer: Debian Security Tools <team+pkg-security@tracker.debian.org>
+Installed-Size: <size-in-kb>
```

Comment:
I've just spotted this, I can't find the place where this is replaced.

Reply:
Seems like it is not mandatory. You can remove it if you like.

Reply:
Oh the Installed size gets updated when building in the Docker Container.

Reply:
![image](https://github.com/user-attachments/assets/6345c1c5-619a-4417-b086-b82ccde28745)